app/backend/src/couchers/email/__init__.py

```diff
@@ -56,6 +59,8 @@ def _render_email(template_file, template_args={}):
 
     The body is run through twice, once for the plaintext, once for HTML, and then the HTML version is run through
     render_html above to turn it into HTML.
+
+    If is_system_notification is set, it'll be rendered on a separate system template.
```

Comment:
What should I do instead?

Reply:
Use `plain_base_template`. Or will it break?

Reply:
No, it's fine... But it just annoys me when the emails come in to the ticket system, etc with that annoying and inaccurate footer.

Reply:
Which footer, you mean:

> You're receiving this email because you signed up for Couchers.org. If you have any issues, email us at <support@couchers.org>.

Reply:
Yeah, that one

Reply:
Live with it :) Rather than having to maintain two highly untested plain/html template files.

I was going to suggest moving the footer to the markdown templates, but I think we broke it out to be able to do a fancy HTML footer.

Reply:
Okay... 😢